DMTN-272.tex

```diff
@@ -49,7 +50,184 @@
 % ADD CONTENT HERE
 % You can also use the \input command to include several content files.
 
+\section{Architecture}
+So far, we have run the experiments with three different architectures:
+
+\begin{itemize}
+\item{An off-the-shelf architecture based on ResNet50~\citep{he2016deep}.}
+\item{An off-the-shelf architecture based on VGG6~\citep{simonyan2014very}.}
+\item{A custom architecture based on the encoder part of TransiNet, internally referred to as rbTN~\citep{sedaghat2018effective}.}
+\end{itemize}
+
+The former two (ResNet50 and VGG6) have been more extensively tested.
+
+\section{Data}
+We used the version of the DC2 dataset available as a butler repository, as the main source of our data.
+
+We have chosen the tract 3080 for training and 4024 for test/validation. Reasoning TBA.
+Figures~\ref{fig:exposure_hist}~,~\ref{fig:detector_hist} show the distributions of exposure ($\simeq$visit) and detector numbers in the raw images currently captured (simulated) in each of the chosen tracts. The detector indices are, neglecting the narrow peaks, roughly uniformly distributed, meaning that we have an almost equal number of raw images (roughly 300) for each detector/CCD in each of the chosen tracts. This is particularly important for the training phase as we do not want the model to be biased towards specific detector characteristics.
+
+More details about the exact data-set used for each experiment is included in the experiment's subsection.
+
+\begin{figure}[h!]
+  \centering
+  \includegraphics[width=0.7\textwidth]{exposure_hist.png}
+  \caption{Distribution of exposure numbers of the existing raw images of each of the tracts. Left: tract \#3080, right: tract \#4024.}
+  \label{fig:exposure_hist}
+\end{figure}
+
+\begin{figure}[h!]
+  \centering
+  %\includegraphics[width=0.7\textwidth]{tract_detectors.png}
+  \includegraphics[width=0.7\textwidth]{detector_hist.png}
+  \caption{Distribution of the detector indices corresponding to the raw images of each of the tracts. Left: tract \#3080, right: tract \#4024.}
+  \label{fig:detector_hist}
+\end{figure}
+
+\clearpage
+Template images were created per-tract, in the sense that the data input to the APTemplate pipeline was constrained to a single tract at each run. Figure~\ref{fig:tract_templates} illustrates an exemplar instance, where the effect of limiting the raws to a single tract apperas as croppings in the output templates. This is not of a concern in this very application though, since the downstream tasks receive proper masks and will generate cutouts only of the regions from which enough informative data can be extracted.
```

Comment:
```suggestion
Template images were created per-tract, in the sense that the data input to the APTemplate pipeline was constrained to a single tract at each run. Figure~\ref{fig:tract_templates} illustrates an exemplar instance, where the effect of limiting the raws to a single tract appears as croppings in the output templates. This is not of a concern in this very application though, since the downstream tasks receive proper masks and will generate cutouts only of the regions from which enough informative data can be extracted.
```